Update no-consumers-consuming doc

1pretz1 opened the pull request:
Updates documentation. We've also decided to change the name of the doc
to be more reflective of what it actually is describing.

Trello:
https://trello.com/c/0OHZT3cq/458-investigate-rabbitmq-and-improve-the-documentation

benthorner commented on source/manual/alerts/rabbitmq-high-number-of-unprocessed-messages.html.md:
```diff
@@ -0,0 +1,45 @@
+The Icinga check is performed by connecting to RabbitMQ's admin API and
+triggering if the number of messages on the queues listed above crosses certain
+thresholds: 25 for critical and 5 for warning. These thresholds are set in
+[puppet][thresholds].
```
Minor: I usually try to link to the code at a specific commit, so that the link still works if the content changes.

1pretz1 replied:
Sure 👍

benthorner commented on source/manual/alerts/rabbitmq-high-number-of-unprocessed-messages.html.md:
```diff
@@ -0,0 +1,45 @@
+
+We check that there is not a significant build up of messages on certain queues
+as we expect messages to be processed quickly. The queues this alert applies to
+are: `email_alert_service`, `email_unpublishing` and `cache_clearing_service-*`
```
Could we add a link to verify this?

1pretz1 replied:
Sure, I've added links to the config for each queue.

benthorner commented on source/manual/alerts/rabbitmq-high-number-of-unprocessed-messages.html.md:
```diff
@@ -0,0 +1,45 @@
+
+## Troubleshooting
+
+For troubleshooting steps, see [here][toubleshooting_steps].
```
Broken link here.

1pretz1 replied:
Good spot

benthorner commented on source/manual/alerts/rabbitmq-high-number-of-unprocessed-messages.html.md:
```diff
@@ -0,0 +1,45 @@
+thresholds: 25 for critical and 5 for warning. These thresholds are set in
+[puppet][thresholds].
+
+## Consequences
```
I don't think we need this section - we've said all this already.

benthorner reviewed (commented):
Good to have something specific to this alert 👍. Might be helpful to link/compare against the other one, in case they all happen at the same time? I think we can make this a little shorter, too.

1pretz1 commented:
Thanks for the review @benthorner, I've addressed your comments.

Also first time around I forgot to mention the cache_clearing_service's queue thresholds, I've updated the doc accordingly 👀

benthorner commented on source/manual/alerts/rabbitmq-high-number-of-unprocessed-messages.html.md:
```diff
@@ -0,0 +1,49 @@
+
+> **Note**
+>
+> If there is a build up of messages, you may see the [no consumers
```
```suggestion
> You may see the [no consumers
```

When I read this, it sounded a bit like egg -> chicken -> egg.

benthorner reviewed (approved):
Nice one 🇹🇱
